android/app/src/main/java/org/fossasia/openevent/utils/DateUtils.java

```diff
@@ -173,4 +173,9 @@ public static String getRelativeTimeFromTimestamp(String timeStamp) throws Parse
                 (timeCreatedDate.getTime()),
                 System.currentTimeMillis(), android.text.format.DateUtils.SECOND_IN_MILLIS);
     }
+
+    public static boolean isUpcomingSession(Date start, Date end, Date current) {
```

Comment:
@iamareebjamal Okay, I'll add a new class for this then. Also, I'll also change the name for this class to dateConversion so that it doesn't confuse new developers..